assets/src/dashboard/app/views/shared/pageHeading.js

```diff
@@ -67,6 +82,14 @@ const SearchInner = styled.div`
   width: min(190px, 100%);
 `;
 
+const HeadingBodyWrapper = styled(BodyWrapper)`
+  display: flex;
```

Comment:
The reason they are not consolidated is the bottom-border actually has a 10px gap. It doesn't go edge-to-edge. So we use a parent container to create a 10px padding. If we used margins, then when content scrolls up we'd see 10px "peak out" from the sides. Alternatively, we could collapse this by creating a 1px tall rule to serve as our border with 10px margins.

Reply:
ah, i see. i think i'd prefer the parent container because it's easier to read through? Fewer chances of a new person on the team seeing a 1px tall rule and "investigating" why?